airflow/utils/setup_teardown.py

```diff
@@ -128,8 +128,6 @@ def _set_dependency(task, receiving_task, upstream):
 
     @classmethod
     def update_context_map(cls, task: DependencyMixin):
-        from airflow.models.abstractoperator import AbstractOperator
```

Comment:
I think this needs to be kept since `AbstractOperator` is only imported for typing globally.

Reply:
oops